docs/Configuration/yaml-files.md

```diff
@@ -492,8 +497,9 @@ org_settings:
 
 ### integrations
 
-The `integrations` section lets you define calendar events and ticket settings for failing policy and vulnerability automations. Learn more about automations in Fleet [here](https://fleetdm.com/docs/using-fleet/automations).
-In addition, you can define the SCEP proxy settings for Network Device Enrollment Service (NDES). Learn more about SCEP and NDES in Fleet [here](https://fleetdm.com/guides/ndes-scep-proxy).
+The `integrations` section lets you configure your Google Calendar, Jira, and Zendesk. After configuration, you can enable automations like calendar event and ticket creation for failing policies and vulnerabilities. Currently, enabling automations is only available using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML files coming soon). Learn more about automations in Fleet [here](https://fleetdm.com/docs/using-fleet/automations).
```

Comment:
You can enable some automations via YAML ([example](https://fleetdm.com/docs/configuration/yaml-files#separate-file)). Or is this about a different kind of automation? (If so, I think we need to make that language clearer.)

Reply:
Ah, good catch. I think it's specifically ticket creation that you can't enable via YAML. I updated this section to the following:

_The `integrations` section lets you configure your Google Calendar, Jira, and Zendesk. After configuration, you can enable automations like calendar event and ticket creation for failing policies and vulnerabilities. Learn more about automations in Fleet [here](https://fleetdm.com/docs/using-fleet/automations).

Currently, enabling ticket creation is only available using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML files coming soon)._